youtube_dl/extractor/gamestar.py

```diff
@@ -9,7 +9,7 @@
 
 
 class GameStarIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?gamestar\.de/videos/.*,(?P<id>[0-9]+)\.html'
+    _VALID_URL = r'https?://(?:www\.)?game(pro|star)\.de/videos/.*,(?P<id>[0-9]+)\.html'
```

Comment:
Don't capture groups you don't use. Add a test.

Reply:
Did both. Also fixed view_count and comment_count.